bqskit/ir/lang/qasm2/visitor.py

```diff
@@ -261,6 +256,9 @@ def fill_gate_defs(self) -> None:
         self.gate_defs['rccx'] = GateDef('rccx', 0, 3, RCCXGate())
         self.gate_defs['rc3x'] = GateDef('rc3x', 0, 4, RC3XGate())
 
+        # reset
+        self.gate_defs['reset'] = GateDef('reset', 0, 1, Reset())
```

Comment:
Reset and measurement are special instructions in the qasm 2.0 language and shouldn't be handled like a gate. See the comments below, remove this.